src/items/repositories/item.repository.ts

```diff
@@ -53,14 +57,60 @@ export class ItemRepository extends Repository<Item> {
   }
 
   async findByAccount(accountId: string): Promise<Item[]> {
-    return this.getItemQueryBuilder()
+    return this.createQueryBuilder('item')
+      .select([
```

Comment:
Whay aren't you using the querybuilder? that way we will reuse a ton of repeated code